JustSaying/AwsTools/MessageHandling/SqsQueueByNameBase.cs

```diff
@@ -74,26 +74,28 @@ public virtual async Task<bool> CreateAsync(SqsBasicConfiguration queueConfig, i
                 if (ex.ErrorCode == "AWS.SimpleQueueService.QueueDeletedRecently")
                 {
                     // Ensure we wait for queue delete timeout to expire.
-                    _log.LogInformation($"Waiting to create Queue due to AWS time restriction - Queue: {QueueName}, AttemptCount: {attempt + 1}");
+                    _log.LogInformation("Waiting to create queue '{QueueName}', due to AWS time restriction. Attempt number {attemptCount}.",
```

Comment:
Maybe add into the message that it's going to wait a minute before trying again?